### PR TITLE
fix: clang format action

### PR DESCRIPTION
**Pull request by xiaoyifang**

### CODE REVIEW

**Comment by shenlebantongying** [comment]
We should specify binary path by `git-clang-format --binary=/home/linuxbrew/.linuxbrew/opt/clang-format/bin/clang-format`

File: .github/workflows/auto format.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
         run: |
           eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
           brew install clang-format
-          (git-clang-format --binary=/home/linuxbrew/.linuxbrew/opt/clang-format/bin/clang-format --style=file HEAD^) || true
+          (git-clang-format --binary=/home/linuxbrew/.linuxbrew/opt/clang-format/bin/clang-format --binary=/home/linuxbrew/.linuxbrew/opt/clang-format/bin/clang-format --style=file HEAD^) || true
```
Comment (xiaoyifang) [inline .github/workflows/auto format.yml:30]: double --binary

**Review by shenlebantongying — CHANGES_REQUESTED** [review]

File: .github/workflows/auto format.yml (hunk 1 above)
Comment (shenlebantongying) [reply .github/workflows/auto format.yml]: :)

Comment (shenlebantongying) [reply .github/workflows/auto format.yml]: I am blind

**Pull request merged**